zerver/management/commands/delete_user.py

```diff
@@ -0,0 +1,43 @@
+from argparse import ArgumentParser
+from typing import Any
+
+from zerver.lib.actions import do_delete_user
+from zerver.lib.management import CommandError, ZulipBaseCommand
+from zerver.models import UserProfile
+
+
+class Command(ZulipBaseCommand):
+    help = (
+        "Delete a user, including all messages sent by them and personal messages received by them."
```

Comment:
We probably should add a bit more of a warning in the help text that we recommend deactivating users for cases where someone has left the organization but you want to preserve their messages history; and that deleting a user makes it as though that user never existed.  (So emoji reactions, private messages and group private messages involving them, will all be deleted, which can be disruptive)